### ISSUE TITLE
selecting current day in global reports page changes data to 01 January 1970

### ISSUE THREAD

**Issue by anugcodes** [InternetHealthReport/ihr-website#350]
**Describe the bug**
In the Global Report page selecting the current date thought the Calendar changes the date to 01 January 1970.
**To Reproduce**

1. Go to 'http://localhost:8080/ihr/en-us/global_report'
2. Click on Calendar icon below the Global Report heading.
3. reSelect the current day (19th March 2023 for me)
4. Global Report for the date '01 January 1970' will be loaded.

**Expected behavior**
On re selecting the current date, it should only close the calendar without making any network request or anything.

**Screenshots**
![first issue](https://user-images.githubusercontent.com/110781959/226174028-1bb20a45-4297-4b6d-a72d-626ca8f3baf0.jpg)

**Desktop (please complete the following information):**

- OS: [e.g.  Windows 10]
- Browser [e.g. Firefox Browser]
- Version [e.g. 111.0 (64-bit)]

**Additional context**
i found that this issue occurs upon reselecting current date on the calendar for any date value.

**Comment by anugcodes** [comment]
Hey @AhnafSulficar  you will have to contact @romain-fontugne  or @Aniket762  if you want to work on this.

**Comment by kkartik07** [comment]
@anugcodes Are you working on this issue or I can work on this?

**Comment by kkartik07** [comment]
@anugcodes Actually this is happening whenever we select the same date twice, i.e. if current date is 22 March chose already and we try to choose again the same date by clicking on 22 March from date picker, then sets the date to Jan 1 1970.

**Comment by Rithvik-padma** [comment]
@kkartik07 @anugcodes I think this problem doesn't persist in the deployed website.

**Comment by kkartik07** [comment]
@Rithvik-padma No, its there on the deployed site as well, you can verify by re-clicking on the same date.

**Comment by romain-fontugne** [comment]
Thanks for pointing this out. I also saw that problem.

**Comment by romain-fontugne** [comment]
@kkartik07 go for it!

**Comment by kkartik07** [comment]
Can I know the meaning of this mask?
`const QTIME_MASK = 'YYYY-MM-DDTHH:mm:ss.SSSZ'`

I have checked official docs of quasar qdate, and the mask options of which examples are given are :
![image](https://user-images.githubusercontent.com/87076033/230308251-77be1ae0-8ad3-42de-8206-abad02241eec.png)

After I have used one of them, the issue is getting solved and functionality is also proper but the date value binding isn't happening in the date picker, so the date is not being shown inside date picker.
![image](https://user-images.githubusercontent.com/87076033/230308794-0d496338-40b4-4c43-a4d8-266880868644.png)

There is one more solution to that, hiding the date by using `minimal` attribute for qdate, which does not show the date value inside the date picker. 
![image](https://user-images.githubusercontent.com/87076033/230309129-79241f70-150c-4a5c-aa00-c496184ca964.png)

Is it okay if we change it like this?

**Comment by kkartik07** [comment]
@anugcodes ok, great!
